### DISCUSSION TITLE
I made a cool thing with Tidy3D

### DISCUSSION THREAD

**Discussion by tylerflex** [flexcompute/tidy3d#120] [Show and tell]
Here's my cool thing!

**Comment by joamatab** [comment]
The [tidy3d plugin](https://gdsfactory.github.io/gplugins/) for  [gdsfactory](https://gdsfactory.github.io/gdsfactory/index.html) allows you to define your layout once and be able to get simulation results from tidy3d

**Comment by mustafacc** [comment]
[minimal standalone gds-to-tidy3d module]( https://github.com/mustafacc/gds_tidy3d/)

adding tests in progress..
